app/src/main/res/values/strings.xml

```diff
@@ -173,4 +174,15 @@
     <string name="turn_gps_on">Turn on your GPS to see events near you</string>
     <string name="activate_gps">ACTIVATE GPS</string>
     <string name="account_settings">Account settings</string>
+
+    <!-- Polls and Voting -->
+    <string name="polls_topbar_title">Polls and Voting</string>
+    <string name="polls_legend_text">Event Polls</string>
+    <string name="polls_empty_poll_list">There were no polls created for this event</string>
```

Comment:
"No polls created for this event" is shorter

Reply:
Less to read is better.